README.md

```diff
@@ -5,7 +5,6 @@ languages:
 - bicep
 products:
 - azure
-- deployment-environments
```

Comment:
This is for ADE (Azure Deployment Environments).  What is invalid about it?

Reply:
OK - so should we move it into a `services` section then?

Reply:
@hemarina Any ideas?